articles/types-of-styling-in-css/index.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/types-of-styling-in-css/
+title: Types of CSS (Cascading Style Sheets)
+description: This article will go over the different styling options in CSS, how to apply them, and when best to implement each style.
+author:
+date: 2020-11-02T00:00:00-13:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/types-of-styling-in-css/hero.jpg
+    alt: Types of CSS (Cascading Style Sheets) example image
+
+---
+We will be going over Cascading Style Sheets commonly known as CSS. CSS is what turns your everyday markup language, HTML, into a one of a kind expression that leaves people in admiration. CSS complements the elements in HTML to help come up with various designs that make a website pop. From background colors, font-size, font-weight, font-family, and much more the design of a website becomes limited by only your imagination.
+<!--more-->
+CSS is the main playground for any developer. To use an analogy, CSS is like all the spices you add to your food to give it a rich, enticing aroma and the desired taste when you finally present it. However, just throwing things in there randomly doesn't guarantee greatness.
+
+You still have to know what goes with what, when to apply a specific style or when something just doesn't fit in. How you implement your CSS can either make or break your website. Below we'll be looking at the different styling options in CSS, how to apply them, and when best to implement each style.
+
+### Styling in CSS
+There are 3 distinct methods for styling in CSS. Each level of styling is given a different hierarchical priority and is used for different reasons.
+
+The 3 methods are further grouped into two categories. Namely *Internal CSS* and *External CSS*. *Internal CSS* is comprised of Local Styles and Page-Level Styles while *External CSS* is just as the name implies.
+
+1. **Local styles:** Also known as inline. This form is defined within your HTML tags/elements. It is mostly used to style specific elements in your code.
+
+```HTML
+  <html>
+    <head>
+      <title>Cascading Style Sheets</title>
+    </head>
+    <body>
+      <p style = "font-family: sans-serif;
+          font-size: 1.2em
+          font-style: italic;">
+          This paragraph is an example of a local style.
+        </p>
+        <p>This
+    </body>
+  </html>
+```
+
+By using the style tag in the HTML element *p*, we're able to change the font, font-size, and style only for the first *p*. As for the second *p* element it retains its default style.
```

Comment:
This sentence was odd to me - but again it may be because the reader needs prior knowledge on HTML elements to follow along.